Makefile

```diff
@@ -284,9 +284,7 @@ ifeq ($(LLVM_BUILD),$(S2E_BUILD))
 
 stamps/clang-binary: $(CLANG_BINARY) | stamps
 	tar -xmf $<
-	mkdir -p $(S2E_PREFIX)
-	cp -r $(CLANG_BINARY_DIR)/* $(S2E_PREFIX)
-	rm -r $(CLANG_BINARY_DIR)/*
+	cp -r $(CLANG_BINARY_DIR) $(S2E_PREFIX)
```

Comment:
Not clear why this change is needed. It looks unrelated to the commit, so either remove it or put it in a separate commit.

Reply:
This is not needed.